plugins/Live/templates/getLastVisitsStart.twig

```diff
@@ -2,11 +2,13 @@
 {% set maxPagesDisplayedByVisitor=100 %}
 
 <ul id='visitsLive'>
-    {% for visitor in visitors %}
+    {% for visitor in dataTable.getRows() %}
+        {% set idSite = visitor.idSite %}
```

Comment:
Yes this is because now it is  visualization and thus there is no `visitors` variable but `dataTable`.